### PR TITLE
Add harbor ova install guide

### PR DESCRIPTION
**Pull request by stonezdj**
Add harbor ova install guide

### CODE REVIEW

File: docs/install_guide_ova.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+   If you deployed the appliance so that it obtains its address via DHCP, go to the **Summary** tab for the appliance VM and note the address.
+
+8. (Optional) If you provided a static network configuration, view the network status of the appliance.
+
+    1. In the **Summary** tab for the appliance VM, launch the VM console
+    2. In the VM console, press the right arrow key. 
+
+    The network status shows whether the network settings that you provided during the deployment match the settings with which the appliance is running. If there are mismatches, power off the appliance and select **Edit Settings** > **vApp Options** to correct the network settings.
+    
+9. In a browser, go to  https://<i>harbor_appliance_address</i> and when prompted, enter the username admin and the password of admin input in step 4
```
Comment (wy65701436) [inline docs/install_guide_ova.md]: default is the FQDN name. do you mean the harbor_appliance_address is the FQDN?

File: docs/install_guide_ova.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    **IMPORTANT**: If you set a static IP address for the appliance, use spaces to separate DNS servers. Do not use comma separation for DNS servers. 
+
+    - Leave the text boxes blank to use auto-generated certificates.
+   
+7. When the deployment completes, refresh the current page and power on the appliance VM.
+   ![Screenshot of power on](img/ovainstall/poweron.png)
+   Press Enter to login
+   ![Screenshot of login root](img/ovainstall/poweron2.png)
+
+   Login to the console with root username and password, and get the harbor appliance address by ifconfig.
```
Comment (wy65701436) [inline docs/install_guide_ova.md]: just get the FQDN from the summary page, no need to login to the OVA.

File: docs/install_guide_ova.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+6. Expand **Networking Properties** and optionally configure a static IP address for the appliance VM. 
+
+    To use DHCP, leave the **Networking Properties** blank.
+
+    **IMPORTANT**: If you set a static IP address for the appliance, use spaces to separate DNS servers. Do not use comma separation for DNS servers. 
+
+    - Leave the text boxes blank to use auto-generated certificates.
+   
+7. When the deployment completes, refresh the current page and power on the appliance VM.
```
Comment (wy65701436) [inline docs/install_guide_ova.md]: It will take several minutes after power on as it needs to load docker images.

Comment (stonezdj) [reply docs/install_guide_ova.md]: Add this info

File: docs/install_guide_ova.md (hunk 2 above)
Comment (stonezdj) [reply docs/install_guide_ova.md]: Remove login 
Add:
Go to the **Summary** tab for the appliance VM and note the DNS Name.

File: docs/install_guide_ova.md (hunk 1 above)
Comment (stonezdj) [reply docs/install_guide_ova.md]: Change to DNS Name

**Pull request merged**